.gitlab-ci.yml

```diff
@@ -21,7 +21,7 @@ include:
   - .gitlab/deps_build/deps_build.yml
   - .gitlab/deps_fetch/deps_fetch.yml
   - .gitlab/dev_container_deploy/include.yml
-  - .gitlab/e2e/e2e.yml
+  - .gitlab/e2e/include.yml
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏IMO `e2e.yml` is more explicit than `include.yml`

Reply:
This [file](https://github.com/DataDog/datadog-agent/pull/28145/files#diff-d6df45273d5118c5ea01611a014c7391d4ff0672d89190299fe10be8a46a6ba2) is only listing `includes` that's why (and it's the same naming convention above)

Reply:
By convention, if a stage has multiple config files, we put all these in an include block in a `include.yml` file for the stage.